Library/Formula/gnu-complexity.rb

```diff
@@ -0,0 +1,41 @@
+class GnuComplexity < Formula
+  desc "Measures complexity of C source"
+  homepage "https://www.gnu.org/software/complexity"
+  url "http://ftpmirror.gnu.org/complexity/complexity-1.5.tar.xz"
+  mirror "https://ftp.gnu.org/gnu/complexity/complexity-1.5.tar.xz"
+  sha256 "1f5194c0dc6e813ea1dc7f36bfd05f15786a8ad6e9b3ab65b53d2263a0d93102"
+
+  depends_on "autogen"
+
+  def install
+    system "./configure", "--disable-debug",
```

Comment:
It seems to be? When I built it locally I didn't see the configure script complain. If you see something different, let me know.

This is also a newer version.